Adapt libinit to Android 11

NurKeinNeid opened the pull request.

TheScarastic committed:
msm8998-common: libinit: include full static libbase

Change-Id: Icb67cd6b6759d8cc4fd2e7a98cea74ac68e8ad07
Signed-off-by: NurKeinNeid <mralexman3000@gmail.com>

wight554 committed:
msm8998-common: init: Replace property_set() with android::base::SetProperty()

* follow up https://android.googlesource.com/platform/system/core/+/c88d8f93cfaa8cbf368a0fa94e8bc84bc55c3ece

Signed-off-by: NurKeinNeid <mralexman3000@gmail.com>

poad42 commented:
Thanks Alex! This presumably also needs a system/core for vendor libinit? Will pick that, pull this and test.

NurKeinNeid commented:
Yes, I also added a PR for that: https://github.com/yaap/system_core/pull/2

> Thanks Alex! This presumably also needs a system/core for vendor libinit? Will pick that, pull this and test.

Yes, I also added a PR for that: https://github.com/yaap/system_core/pull/2
